examples/auth-chakra-ui/src/components/table/columnSorter.tsx

```diff
@@ -17,14 +18,16 @@ export const ColumnSorter: React.FC<ColumnButtonProps> = ({ column }) => {
             onClick={column.getToggleSortingHandler()}
             style={{
                 transition: "transform 0.25s",
```

Comment:
Also don't need this anymore. actually, we can remove `style` completely 🥹

```suggestion
                transition: "transform 0.25s",
```

Reply:
removed `style` completely. Please review.

Reply:
thanks 🎉